CHANGELOG.md

```diff
@@ -59,6 +59,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - #3865 - Added a new nightly icon and app name
 
 ### Changed
+- #3278 - Updates strings in preferences
```

Comment:
We really need to restructure our Changelog. It's an arbitrarily-ordered list right now that's very hard to read. We should just start over with a date-based list or one entry per major release.

Reply:
@colintheshots ahah, I had the same thought when I added this line.